regex/definitions.go

```diff
@@ -81,6 +81,12 @@ var DefinitionReferenceRegex = regexp.MustCompile(`{{([a-zA-Z0-9-_]+)}}`)
 // The version declared on the file is captured in group 1.
 var CRSVersionRegex = regexp.MustCompile(`^(# OWASP (ModSecurity Core Rule Set|CRS) ver\.)(.+)$`)
 
+// ShortCRSVersionRegex matches the version contained on every rules file.
+// The version declared on the file is captured in group 1.
+// This regex is used to match the version in the setup file.
+// As numbers can grow to something like 10.12.123, the range is set to 3-8 to capture bigger numbers.
+var ShortCRSVersionRegex = regexp.MustCompile(`(setvar:tx.crs_setup_version=)(\d+)`)
+
 // CRSCopyrightYearRegex matches the version and year range of the copyright text in setup,
 // setup example, and rule files.
 // The matched end year of the copyright year range will be captured in group 1.
```

Comment:
In all the regular expressions below, the year / version is actually in capture group 2, not 1. Could you update the comments?